__tests__/test-data.test.js

```diff
@@ -0,0 +1,53 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("api/topics", () => {
+  test("recieve status code 200", () => {
+    return request(app).get("/api/topics").expect(200);
+  });
+  test("returns an array ", () => {
```

Comment:
Could this not have been combined into one test? We can chain expecting a 200 status code and then if it is an array containing the objects we are expecting.